learning_journal_db/templates/layout.jinja2

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href='https://fonts.googleapis.com/css?family=PT+Serif|Montserrat' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="../static/base.css">
```

Comment:
This right here is why your `edit-entry.jinja2` isn't working. You're using a relative file path, but your edit entry renderings are 1 level deeper than your other pages. Thus, this breaks. **FOR ALL URLS AND STATIC FILES, EITHER USE** `request.route_url` **OR** `request.static_path`